05-policy-changes/policy-example.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    REQUEST_URL = 'https://api.contextual.ai/v1'
</pre><p>we can remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I like to keep the request URL in case people are working on a different instance of Contextual AI than our cloud.